.github/workflows/test-reporter.yml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
I have tried this action with all the permissions, and I did not succeed to make it work, it is true that it was in a private repository.

Reply:
Good to know, I somehow thought it was the one in place, so let's try if it works, since `workflow_run` only works if it has been added previously.